My Series List/My Series List/Model/Repository/RepositoryController.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  RepositoryController.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 19/11/2021.
+//
+
+import Foundation
+
+class RepositoryController {
```

Comment:
This class should be simply renamed as Repository. In this case it should be SeriesRepository